applications/GeoMechanicsApplication/custom_elements/interface_element.cpp

```diff
@@ -189,30 +202,30 @@ int LineInterfaceElement::Check(const ProcessInfo& rCurrentProcessInfo) const
     return 0;
 }
 
-Element::DofsVectorType LineInterfaceElement::GetDofs() const
+Element::DofsVectorType InterfaceElement::GetDofs() const
 {
     const auto no_Pw_geometry_yet = Geometry<Node>{};
     return Geo::DofUtilities::ExtractUPwDofsFromNodes(GetGeometry(), no_Pw_geometry_yet,
                                                       GetGeometry().WorkingSpaceDimension());
 }
 
-std::vector<Matrix> LineInterfaceElement::CalculateLocalBMatricesAtIntegrationPoints() const
+std::vector<Matrix> InterfaceElement::CalculateLocalBMatricesAtIntegrationPoints() const
 {
     const auto& r_integration_points = mIntegrationScheme->GetIntegrationPoints();
     const auto  shape_function_values_at_integration_points =
         GeoElementUtilities::EvaluateShapeFunctionsAtIntegrationPoints(r_integration_points, GetGeometry());
 
     auto result = std::vector<Matrix>{};
     result.reserve(shape_function_values_at_integration_points.size());
-    auto calculate_local_b_matrix = [&r_geometry = GetGeometry(), p_policy = mStressStatePolicy.get()](
-                                        const auto& rShapeFunctionValuesAtIntegrationPoint,
-                                        const auto& rIntegrationPoint) {
+    auto calculate_local_b_matrix = [&r_geometry = GetGeometry(), p_policy = mpStressStatePolicy.get(),
```

Comment:
If we don't need to, I'd prefer to just pass the member instead of dealing with the raw pointer (but I'm not sure if it compiles):
```suggestion
    auto calculate_local_b_matrix = [&r_geometry = GetGeometry(), &mpStressStatePolicy,
```

Reply:
It is possible to removed it from the list because there is `this` now.